src/riak_cs_console.erl

```diff
@@ -183,10 +185,119 @@ cleanup_orphan_multipart(Timestamp) when is_binary(Timestamp) ->
     _ = io:format("~nAll old unaborted orphan multipart uploads have been deleted.~n", []).
 
 
+-spec resolve_siblings(binary(), binary()) -> any().
+resolve_siblings(RawBucket, RawKey) ->
+    {ok, RcPid} = riak_cs_riak_client:start_link([]),
+    try
+        {ok, Pid} = case RawBucket of
+                        <<"0o:", _/binary>> ->
+                            riak_cs_riak_client:manifest_pbc(RcPid);
```

Comment:
This does not work for multibag environment because riak_cs_riak_client does not know which bag it should use for manifest.